docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -176,6 +176,9 @@ Method | Type | Description
 `withText(text)` | Selector | Creates a selector that filters a matching set by the specified text.
 `withText(re)` | Selector | Creates a selector that filters a matching set using the specified regular expression.
 
+The `withText(text)` method converts the string argument to a regular expression.
```

Comment:
> The `withText(text)` method converts the string argument to a regular expression.

It's not true. We should say only that `withText(text)` it's not about an element's text content is strict equal to `text` but element's text content contains `text`